### ISSUE TITLE
typo: add missing `|` to command list

### ISSUE THREAD

**Issue by coolaj86** [webinstall/webi-installers#670]
Typo in <https://github.com/webinstall/webi-installers/blob/main/xsv/README.md>:

```diff
 xsv search '^(John|Jane)$' --select 'First Name' ./address-book.csv |
     xsv select 'ID,First Name,Last Name' |
     xsv sort --select 'Last Name,First Name' |
-    xsv slice -s 0 -n 5
+    xsv slice -s 0 -n 5 |
     xsv table
```

**Comment by starvader13** [comment]
I would love to take this issue.

**Comment by coolaj86** [comment]
Closed via #671
